### PR TITLE
Touch ups

### PR DESCRIPTION
**Pull request by MichaelAldrich**
A few things that were easy to do

### CODE REVIEW

File: nxt_editor/main_window.py [hunk 1]
```diff
@@ -1335,29 +1338,26 @@ def window_action_triggered(self, action=None):
...
+        QtGui.QDesktopServices.openUrl(
+            QtCore.QUrl.fromLocalFile(USER_PLUGIN_DIR)
+        )
+
+    @staticmethod
+    def open_logs_dir():
+        try:
+            log_dir = nxt_log.LOG_DIR
+        except AttributeError:
+            # Guess the log dif if nxt core is old.
```
Comment (ImLucasBrown) [inline nxt_editor/main_window.py]:
```suggestion
            # Guess the log dir if nxt core is old.
```

File: nxt_editor/node_graphics_item.py [hunk 2]
```diff
@@ -452,7 +452,8 @@ def draw_title(self, painter, lod=1.):
 
         # draw collapse state arrow
         for arrow in self.collapse_arrows:
-            self.scene().removeItem(arrow)
+            if arrow.scene():
+                self.scene().removeItem(arrow)
```
Comment (ImLucasBrown) [inline nxt_editor/node_graphics_item.py:456]: I assume we can be sure the arrow's scene is _always_ the graphic item's scene?

File: nxt_editor/stage_view.py [hunk 3]
```diff
@@ -1281,7 +1281,8 @@ def remove_node_graphic(self, node_path):
         self.remove_node_connection_graphics(node_path)
 
         def handle_del():
-            self.scene().removeItem(graphic)
+            if graphic.scene():
+                self.scene().removeItem(graphic)
```
Comment (ImLucasBrown) [inline nxt_editor/stage_view.py:1285]: Same here.

File: nxt_editor/main_window.py [hunk 4]
```diff
@@ -1335,29 +1338,26 @@ def window_action_triggered(self, action=None):
...
-        elif 'win' in sys.platform:
-            os.startfile(d)
-        else:
-            try:
-                os.system('xdg-open {}'.format(d))
-            except:
-                logger.exception('Failed to open user dir')
+        QtGui.QDesktopServices.openUrl(
+            QtCore.QUrl.fromLocalFile(user_dir.PREF_DIR)
+        )
```
Comment (ImLucasBrown) [inline nxt_editor/main_window.py:1343]: Learn something everyday! 🚀

**Review by ImLucasBrown — APPROVED** [review]
LGTM

File: nxt_editor/node_graphics_item.py (hunk 2 above)
Comment (MichaelAldrich) [reply nxt_editor/node_graphics_item.py:456]: Yes to both, `self.collapse_arrows` will only ever have arrows from the same scene as this node.

**Pull request merged**